packages/kit/src/views/Staking/pages/Stake/index.tsx

```
@@ -32,89 +33,75 @@ function BasicStakePage() {
-    [accountId, networkId, indexedAccountId, token, provider],
-    {
-      revalidateOnFocus: true,
-    },
-  );
-
-  const balanceParsed = tokenResult?.token.balanceParsed || token.balanceParsed;
-  const price = tokenResult?.token.price || token.price;
-
-  const tokenInfo = token.info;
 
   const actionTag = buildLocalTxStatusSyncId({
```

ESLint

[error] Delete `⏎`

(prettier/prettier)